handlers/bongolive/bongolive_test.go

```diff
@@ -46,8 +46,8 @@ var testCases = []ChannelHandleTestCase{
 	{
 		Label:                "Status No params",
 		URL:                  receiveURL,
-		Data:                 "",
-		ExpectedRespStatus:   405,
+		Data:                 "&",
```

Comment:
this wasn't actually testing a POST with no params because Data = "" means use GET